fundingmanager_test.go

```diff
@@ -467,20 +475,19 @@ func recreateAliceFundingManager(t *testing.T, alice *testNode) {
 	}
 }
 
-func setupFundingManagers(t *testing.T, maxPendingChannels int) (*testNode, *testNode) {
-	// We need to set the global config, as fundingManager uses
-	// MaxPendingChannels, and it is usually set in lndMain().
-	cfg = &config{
-		MaxPendingChannels: maxPendingChannels,
-	}
+type cfgOption func(*fundingConfig)
+
+func setupFundingManagers(t *testing.T, maxPendingChannels int,
+	options ...cfgOption) (*testNode, *testNode) {
```

Comment:
If we're going this route, why not also make `maxPendingChannels` a `cfgOption`?

Reply:
Yes, good idea! Done.